app/controllers/AlertController.java

```diff
@@ -49,12 +59,26 @@
  */
 public class AlertController extends Controller {
 
+    private static final String CONFIG_LIMIT = "alerts.limit";
+    private static final String CONFIG_EXECUTIONS_BATCH_SIZE = "alerts.executions.batchSize";
+    private static final String CONFIG_EXECUTIONS_LOOKBACK_DAYS = "alerts.executions.lookbackDays";
```

Comment:
I can't actually see a case where we'd want to tune this to something other than 1, since we really only care about the most recent executions. I figure this would only make sense if you had stale jobs, but even then you'd probably optimize for getting the job to run and back up to date rather than trying to fetch old results.